### PR TITLE
Add Google Crashlytics support for the example app

### PR DESCRIPTION
**Pull request by lukasz-szyszkowski**

### CODE REVIEW

File: Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -497,7 +504,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = C4YGD8Z6G2;
```
Comment (lawrence-forooghian) [inline Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj:507]: Was this change intentional? Don't we want to keep the development team empty and let automatic code signing do its thing?

**Review by lawrence-forooghian — COMMENTED** [review]
@lukasz-szyszkowski is there a reason we've not added Crashlytics to the subscriber example app?

**Comment by lukasz-szyszkowski** [comment]
> @lukasz-szyszkowski is there a reason we've not added Crashlytics to the subscriber example app?

no reason, I'll add a reporter to Subscriber too. Thanks!

File: Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (lawrence-forooghian) [reply Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj:507]: @lukasz-szyszkowski did you have thoughts on this?

Comment (lukasz-szyszkowski) [reply Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj]: here 633c0d8fdfde410438641a29cf9cd01edbbbbcf5

**Review by lawrence-forooghian — APPROVED** [review]

File: Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (lukasz-szyszkowski) [reply Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj]: @lawrence-forooghian Could you resolve this conversation, please?

Comment (lawrence-forooghian) [reply Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj]: Done!
